test/test_fs.py

```diff
@@ -46,21 +48,26 @@ def testfs(tmpdir):
     mnt_dir = str(tmpdir)
     with mp.Manager() as mgr:
         cross_process = mgr.Namespace()
+        need_stop = mgr.Event()
+        cross_process.need_stop = need_stop
         mount_process = mp.Process(target=run_fs,
-                                   args=(mnt_dir, cross_process))
+                                   args=(mnt_dir, cross_process, need_stop))
```

Comment:
need_stop is already an attribute of cross_process, why do you pass it in separately as well?

Reply:
`cross_process.need_stop = need_stop` doesn't work because `Event` type cannot be serialized by this a `Namespace`. I forgot to remove this buggy line ;-)

Reply:
Yeah, but why does it even need to be a mgr.Event()? Just make it a boolean :-).

Reply:
A boolean wouldn't be convenient given `_stop_watcher` has to wait for it to know when it should stop the fuse server.